broadcast/broadcast.go

```diff
@@ -345,6 +336,15 @@ func (broadcast *Broadcast) DelegatorWithdraw(delegator, voter, amount, privKeyH
 	return broadcast.broadcastTransaction(msg, privKeyHex, seq, "")
 }
 
+// ClaimInterest claims interest of a certain user.
+// It composes ClaimInterestMsg and then broadcasts the transaction to blockchain.
+func (broadcast *Broadcast) ClaimInterest(username, privKeyHex string, seq int64) error {
+	msg := model.ClaimInterestMsg{
+		Username: username,
+	}
+	return broadcast.broadcastTransaction(msg, privKeyHex, seq, "")
+}
+
```

Comment:
why move this method without any changes? Am I missing something?

Reply:
No big change. I just saw you move the ClaimInterestMsg from Account module to Vote module, so that I think it would be clearer to put this method close to vote related methods.